### PR TITLE
recycle flutter consoles

### PR DESCRIPTION
**Pull request by devoncarew**
- recycle Flutter consoles (fix https://github.com/flutter/flutter-intellij/issues/478)

@pq

### CODE REVIEW

File: src/io/flutter/console/FlutterConsoleHelper.java [hunk 1]
```diff
@@ -21,60 +22,106 @@
...
+      info.console.attachToProcess(processHandler);
+
+      final MessageView messageView = MessageView.SERVICE.getInstance(project);
+      messageView.runWhenInitialized(() -> {
+        final ContentManager contentManager = messageView.getContentManager();
+        contentManager.setSelectedContent(info.content);
+      });
+    }
+    else {
+      ConsoleView console = createConsole(project, module);
```
Comment (pq) [inline src/io/flutter/console/FlutterConsoleHelper.java]: On the last review I decided not to comment on inconsistent use of `final` but it's distracting enough that I think maybe we should consider re-enabling the inspection that enforces it.  Thoughts?

File: src/io/flutter/console/FlutterConsoleHelper.java [hunk 2]
```diff
@@ -21,60 +22,106 @@
...
+class FlutterConsoleInfo {
+  @NotNull final ConsoleView console;
+  @Nullable final Module module;
+  Content content;
+
+  FlutterConsoleInfo(@NotNull ConsoleView console, @Nullable Module module) {
+    this.console = console;
+    this.module = module;
+  }
+}
```
Comment (pq) [inline src/io/flutter/console/FlutterConsoleHelper.java]: Missing newline.

File: src/io/flutter/console/FlutterConsoleHelper.java (hunk 1 above)
Comment (devoncarew) [inline src/io/flutter/console/FlutterConsoleHelper.java]: sgtm! done

File: src/io/flutter/console/FlutterConsoleHelper.java (hunk 2 above)
Comment (devoncarew) [inline src/io/flutter/console/FlutterConsoleHelper.java]: added

**Pull request merged**